c/validator_utils.h

```diff
@@ -846,22 +846,22 @@ int _load_verification_context(
   global_state_seg.ptr = global_state_buf;
   global_state_seg.size = buf_len;
 
-  uint8_t rollup_version = 0;
+  uint8_t rollup_version = 1;
   if (MolReader_GlobalState_verify(&global_state_seg, false) == MOL_OK) {
     rollup_version =
         *(uint8_t *)MolReader_GlobalState_get_version(&global_state_seg).ptr;
   } else {
-    if (MolReader_GlobalStateV0_verify(&global_state_seg, false) != MOL_OK) {
+    if (MolReader_GlobalStateV1_verify(&global_state_seg, false) != MOL_OK) {
       printf("rollup cell data is not GlobalState format");
       return GW_FATAL_INVALID_DATA;
     }
   }
 
   /* Get block_merkle_root */
   mol_seg_t block_merkle_state_seg;
-  if (0 == rollup_version) {
+  if (1 == rollup_version) {
```

Comment:
We should check the allowed value of rollup_version are `1, 2`

Reply:
Done https://github.com/nervosnetwork/godwoken-scripts/pull/126/commits/34614146b0e4a42af441f59a93b10c2fe7193d52